nncf/quantization/algorithms/weight_compression/openvino_backend.py

```diff
@@ -61,12 +61,10 @@ def is_node_with_weights(node: NNCFNode, graph: NNCFGraph) -> bool:
         return node.layer_attributes and node.layer_attributes.constant_attributes
 
     @staticmethod
-    def get_channel_agnostic_reduction_axes(
-        node_with_weight: NNCFNode, weight_port_id: int, graph: NNCFGraph
-    ) -> Optional[Tuple[int]]:
+    def get_reduction_axes(node_with_weight: NNCFNode, weight_port_id: int, graph: NNCFGraph) -> Optional[Tuple[int]]:
         channel_axes = get_weight_channel_axes(node_with_weight)
         const_shape = node_with_weight.layer_attributes.constant_attributes[weight_port_id]["shape"]
-        return get_channel_agnostic_reduction_axes(channel_axes, const_shape)
+        return get_reduction_axes(channel_axes, const_shape)
```

Comment:
Please, provide results of conformance test for weight compression.

Reply:
As far I know, the build was fixed. You can run it now.

Reply:
build 35 passed